README.md

```diff
@@ -1,2 +1,7 @@
 # InteractiveProgramming
 This is the base repo for the interactive programming project for Software Design, Spring 2016 at Olin College.
+
+To play tic tac toe, just run TicTacToeTest.py.  Moves update every three seconds.  Use a green object to track properly.
```

Comment:
Would also be helpful to include the modules/installs needed to get this up and running.